orchestra/utils/task_lifecycle.py

```diff
@@ -795,7 +795,10 @@ def submit_task(task_id, task_data, iteration_status, worker):
     task.status = next_status
     task.save()
 
-    if task.status == Task.Status.REVIEWING:
+    if task.status == Task.Status.PENDING_REVIEW:
+        # Check the assignment policy to try to assign a reviewer automatically
+        task = _preassign_workers(task, policy_type='reviewer')
```

Comment:
it'd be nice to externalize these string somewhere, like POLICY_TYPE_REVIEWER or something